build.gradle

```diff
@@ -24,8 +25,10 @@ ext {
   configBuildToolsVersion = '23'
   configTargetSdkVersion = 23
 
-  support = [appCompat  : 'com.android.support:appcompat-v7:23.0.0',
-             annotations: 'com.android.support:support-annotations:23.0.0']
+  supportVersion = '23.0.0'
```

Comment:
I tried applying crashlytics to sample app via AS plugin and then switching to this module and it worked fine.